hooks/useContractorInterest.ts

```
@@ -0,0 +1,189 @@
+
+  // Fetch contractors
+  const fetchContractors = useCallback(async () => {
+    setIsLoading(true);
+    setError(null);
+
+    try {
+      const params = {
+        page: currentPage,
+        limit: pageSize,
+        search: debouncedSearch || undefined,
+        emailStatus: filters.emailStatus !== "all" ? filters.emailStatus as any : undefined,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)